resources/views/admin/workedHours/show.blade.php

```diff
@@ -0,0 +1,20 @@
+@extends('layouts.dashboard')
+
+@section('title',__('admin/settings.workedHours'))
+
+@section('content')
+    <dynamic-table :columns="[{
+            name: 'column',
+            label: '@lang('vue.field')',
+            type: 'select',
+            options: {{$workedHoursOptions}},
+            callback: 'numerateOptions'
+        }, {
+            name: 'name',
+            label: '@lang('global.name')',
+        }]" :init-fields="{{collect($workedHours)}}" :sortable="true" action="{{action('Admin\WorkedHoursExportColumnController@create')}}">
+    </dynamic-table>
+    <div class="mt-1">
```

Comment:
maybe add an <hr> between? or even put the export button on top... I think that'll be better